dawn/main/ansible/Ansible.js

```diff
@@ -16,12 +19,31 @@ const StudentCodeStatus = DawnData.StudentCodeStatus;
 
 const runtimeBuilder = ProtoBuf.loadProtoFile(`${protoFolder}/runtime.proto`);
 const RuntimeData = runtimeBuilder.build('RuntimeData');
+const TCPData = runtimeBuilder.build('TCPData');
+
+let test = 0;
```

Comment:
The point of test was a proof of concept that Dawn can react to proto messages without going fully fledged into a fake controller. It'll be removed after more testing verifies that what Dawn does for communication is correct.